src/css/application.css

```diff
@@ -484,6 +484,12 @@ body {
   color: var(--color-red);
 }
 
+.console__entry_inactive * {
```

Comment:
Your instinct to add a prop to `ConsoleOutput` and then use more specific selectors is the right one—unfortunately one of the more annoying patterns required by BEM but overall worth it imo!